source/CMakeLists.txt

```diff
@@ -14,358 +14,259 @@ endif()
+	body/Ship.cpp
+	body/Ship.h
+	body/ShipAICache.cpp
+	body/ShipAICache.h
+	body/Visual.cpp
+	body/Visual.h
 	comparators/ByGivenOrder.h
 	comparators/ByName.h
 	comparators/BySeriesAndIndex.h
-	ship/ShipAICache.cpp
-	ship/ShipAICache.h
+	data/CategoryList.cpp
+	data/CategoryList.h
+	data/CategoryTypes.h
+	data/ConditionSet.cpp
+	data/ConditionSet.h
+	data/ConditionsStore.cpp
+	data/ConditionsStore.h
+	data/Conversation.cpp
+	data/Conversation.h
+	data/CoreStartData.cpp
+	data/CoreStartData.h
+	data/DataFile.cpp
+	data/DataFile.h
+	data/DataNode.cpp
+	data/DataNode.h
+	data/DataWriter.cpp
+	data/DataWriter.h
+	data/File.cpp
+	data/File.h
+	data/Files.cpp
+	data/Files.h
+	data/GameAction.cpp
+	data/GameAction.h
+	data/GameData.cpp
+	data/GameData.h
+	data/GameEvent.cpp
+	data/GameEvent.h
+	data/Gamerules.cpp
+	data/Gamerules.h
+	data/Government.cpp
+	data/Government.h
+	data/Hardpoint.cpp
+	data/Hardpoint.h
+	data/JumpTypes.h
+	data/LocationFilter.cpp
+	data/LocationFilter.h
+	data/Mission.cpp
+	data/Mission.h
+	data/MissionAction.cpp
+	data/MissionAction.h
+	data/NPC.cpp
+	data/NPC.h
+	data/NPCAction.cpp
+	data/NPCAction.h
+	data/News.cpp
+	data/News.h
+	data/Paragraphs.cpp
+	data/Paragraphs.h
+	data/Person.cpp
+	data/Person.h
+	data/Personality.cpp
+	data/Personality.h
+	data/Phrase.cpp
+	data/Phrase.h
+	data/Plugins.cpp
+	data/Plugins.h
+	data/Port.cpp
+	data/Port.h
+	data/PrintData.cpp
+	data/PrintData.h
+	data/RandomEvent.h
+	data/Sale.h
+	data/SavedGame.cpp
+	data/SavedGame.h
+	data/StartConditions.cpp
+	data/StartConditions.h
+	data/SystemEntry.h
+	data/TextReplacements.cpp
+	data/TextReplacements.h
+	data/UniverseObjects.cpp
+	data/UniverseObjects.h
+	engine/AI.cpp
+	engine/AI.h
+	engine/Command.cpp
+	engine/Command.h
+	engine/Date.cpp
+	engine/Date.h
+	engine/Depreciation.cpp
+	engine/Depreciation.h
+	engine/DistanceCalculationSettings.h
+	engine/DistanceCalculationSettings.cpp
+	engine/DistanceMap.cpp
+	engine/DistanceMap.h
+	engine/Distribution.cpp
+	engine/Distribution.h
+	engine/Engine.cpp
+	engine/Engine.h
+	engine/EsUuid.cpp
+	engine/EsUuid.h
+	engine/FireCommand.cpp
+	engine/FireCommand.h
+	engine/Fleet.cpp
+	engine/Fleet.h
+	engine/FleetCargo.cpp
+	engine/FleetCargo.h
+	engine/FormationPattern.cpp
+	engine/FormationPattern.h
+	engine/FormationPositioner.cpp
+	engine/FormationPositioner.h
+	engine/FrameTimer.cpp
+	engine/FrameTimer.h
+	engine/Mortgage.cpp
+	engine/Mortgage.h
+	engine/Outfit.cpp
+	engine/Outfit.h
+	engine/Politics.cpp
+	engine/Politics.h
+	engine/RaidFleet.cpp
+	engine/RaidFleet.h
+	engine/ShipEvent.cpp
+	engine/ShipEvent.h
+	engine/ShipJumpNavigation.cpp
+	engine/ShipJumpNavigation.h
+	engine/ShipManager.cpp
+	engine/ShipManager.h
+	engine/Variant.cpp
+	engine/Variant.h
+	engine/Weapon.cpp
+	engine/Weapon.h
+	image/ImageBuffer.cpp
+	image/ImageBuffer.h
+	image/ImageSet.cpp
+	image/ImageSet.h
+	image/RenderBuffer.cpp
+	image/RenderBuffer.h
+	image/Sprite.cpp
+	image/Sprite.h
+	image/SpriteSet.cpp
+	image/SpriteSet.h
```

Comment:
Putting images in their own directory does make sense. Maybe put this into a separate PR as well?